### PR TITLE
docs: document AWS requirements

### PR DESCRIPTION
**Pull request by mdelapenya**
## What does this PR do?
It adds the AWS requirements that were missing in the docs:

- AWS region where the AMIs are located
- the need of a security group named `e2e` where the remote instances will be created

## Why is it important?
Running the scripts outside Elastic Observability will fail because the security group does not exist. Besides that, the AMIs that we use are located in the Ohio region.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the Unit tests (`make unit-test`), and they are passing locally
- [ ] I have run the End-2-End tests for the suite I'm working on, and they are passing locally
- [ ] I have noticed new Go dependencies (run `make notice` in the proper directory)

## Author's Checklist

- [x] Discovered while testing the scripts with @dikshachauhan-qasource

### CODE REVIEW

File: .ci/README.md [hunk 1]
```diff
@@ -49,6 +49,19 @@ But you must first understand that there are two types of Cloud machines:
...
+#### Running the build scripts outside the Elastic Observability AWS account
+
+In the case you are running the scripts outside the "Elastic Observability" AWS account, please fulfill this requirements before you start creating the instances:
+
+1. Use `us-east-2` (Ohio) as your default AWS region. All the community AMIs that we use are hosted there.
+2. Create a "Security Group" named `e2e`. This security group will allow remote access to certain ports in the remote instances we are creating. In this security group please use `0.0.0.0/0` as the Source for the following ports:
+   - HTTP 80
+   - HTTPS443
+   - SSH 22
+   - Elasticsearch 9200
```
[the remark this answers is not in the document]
Comment (mdelapenya) [reply .ci/README.md:61]: As they are ephemeral, destroyed after the tests are run, they are not secured by default, being accessible from outside. It's possible to SSH into them having a Github username added to the `github-ssh-keys` file.

We are open to any improvement in terms of making them more secure

**Review by dikshachauhan-qasource — APPROVED** [review]
Hi @mdelapenya 

The changes done are looking good to me.

Thanks

File: .ci/README.md [hunk 2]
```diff
@@ -49,6 +49,19 @@ But you must first understand that there are two types of Cloud machines:
...
 2) the VMs running the actual tests, where the Elastic Agent will be installed and enrolled into the stack.
 
+#### Running the build scripts outside the Elastic Observability AWS account
+
+In the case you are running the scripts outside the "Elastic Observability" AWS account, please fulfill this requirements before you start creating the instances:
+
+1. Use `us-east-2` (Ohio) as your default AWS region. All the community AMIs that we use are hosted there.
+2. Create a "Security Group" named `e2e`. This security group will allow remote access to certain ports in the remote instances we are creating. In this security group please use `0.0.0.0/0` as the Source for the following ports:
+   - HTTP 80
+   - HTTPS443
```
Comment (mdelapenya) [inline .ci/README.md:59]:
```suggestion
   - HTTPS 443
```

File: .ci/README.md [hunk 3]
```diff
@@ -26,6 +26,22 @@ It's possible that a consumer of the e2e tests would need to define a specific l
     - **tags**: a Gherkin expression to filter scenarios by tag. It will drive the real execution of the tests, selecting which feature files and/or Cucumber tags will be added to the current test execution. An example could be `linux_integration` or `running_on_beats`. For reference, see https://github.com/cucumber/godog#tags. Required.
     - **platforms**: a list of platforms where the tests will be executed. Valid values are already declared under the `PLATFORMS` object, using the key of the platform as elements in the list. I.e. `["centos8_arm64", "centos8_amd64", "debian_arm64", "debian_amd64", "sles15"]`. Required.
 
+### Adding a new supported platform
```
Comment (mdelapenya) [inline .ci/README.md:29]: @narph I'm interested in your feedback for this particular block

File: .ci/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cachedout) [reply .ci/README.md:77]: @mdelapenya By "accessible" do you mean that just the port is open or is ES running without having authentication enabled (or with a default l/p)?

[the remark this answers is not in the document]
Comment (mdelapenya) [reply .ci/README.md:77]: The ports are open and authentication is there, although credentials are in the code itself

**Review by narph — APPROVED** [review]
LGTM

File: .ci/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mdelapenya) [reply .ci/README.md:77]: @cachedout if you agree, we can continue with this PR and start a discussion on how to create ephemeral and secure instances

File: .ci/README.md [hunk 4]
```diff
@@ -26,6 +26,22 @@ It's possible that a consumer of the e2e tests would need to define a specific l
...
+3. Select an AWS machine type for the new platform, using the existing platforms file as a reference. Our team is actively checking how much cloud resources are consumed by the tests, so please consider using a machine type that already exists.
+ 
+> If you need a machine type that is bigger or powerful, please open a discussion with us so we can understand your needs.
+
+4. Make sure you install the required software dependencies using Ansible. Please take a look at [this file](./ansible/tasks/install_deps.yml). You will find in it the package manager commands and specific filters for the different OS families that are supported by Ansible.
+
+> It's very likely that the new platform is already covered, as there are tasks for Debian/Ubuntu, CentOS (Fedora, RedHat), and Oracle Linux.
+
+5. Add the new supported platform to the test execution, as described above: open the [test descriptor](./.e2e-tests.yaml) and add your new platform to the scenarios you are interested in, as a new platform in the `platforms` array.
```
Comment (mdelapenya) [inline .ci/README.md:43]:
```suggestion
5. Add the new supported platform to the test execution, as described above: open all the test descriptors (i.e. `./.e2e-tests-*.yaml`) and add your new platform to the scenarios you are interested in, as a new platform in the `platforms` array.
```

**Pull request merged**